test-files/cases/attribute/unifying/Config.conf

```diff
@@ -15,5 +15,7 @@
   "review.score.criterion.min.threshold": "0.35",
   "review.score.criterion.invert.thresholding": "true",
   "building.keep.more.complex.geometry.when.auto.merging": "false",
-  "building.outline.update.op.remove.building.relations": "true"
+  "building.outline.update.op.remove.building.relations": "true",
+  "reader.conflate.use.data.source.ids.1": "false",
+  "#": "end"
```

Comment:
Since attribute uses this config file I turned off the new default.  I didn't turn on keeping UNKNOWN2 IDs because some of the datasets have conflicting IDs that cause issues.  That can be fixed later.